sound/soc/sof/topology.c

```diff
@@ -1275,11 +1283,22 @@ static int sof_widget_ready(struct snd_soc_component *scomp, int index,
 	swidget->id = w->id;
 	swidget->pipeline_id = index;
 	swidget->private = NULL;
+	swidget->num_source_pins = 1;
+	swidget->num_sink_pins = 1;
+
+	ret = sof_parse_tokens(scomp, swidget, comp_pin_tokens,
+			       ARRAY_SIZE(comp_pin_tokens), priv->array,
+			       le32_to_cpu(priv->size));
+	if (ret < 0) {
+		dev_err(scomp->dev, "failed to parse component pin tokens for %s\n",
+			w->name);
+		return ret;
+	}
 
-	dev_dbg(scomp->dev, "tplg: ready widget id %d pipe %d type %d name : %s stream %s\n",
+	dev_dbg(scomp->dev, "tplg: ready widget id %d pipe %d type %d name: %s stream %s num_source_pins %d num_sink_pins %d\n",
```

Comment:
can we make this a bit shorter? 129 is a bit over the 100 chars.
```
	dev_dbg(scomp->dev,
		"tplg: ready widget id %d pipe %d type %d name: %s stream %s (pins: %d / %d)\n",
		swidget->comp_id, index, swidget->id, tw->name,
		strnlen(tw->sname, SNDRV_CTL_ELEM_ID_NAME_MAXLEN) > 0 ? tw->sname : "none",
		swidget->num_source_pins, swidget->num_sink_pins);
```
or something?

Reply:
I tried to split the string into two lines, but then I get `quoted string split across lines` from checkpatch, should we keep a long line, or a checkpatch warning?

Reply:
> I tried to split the string into two lines, but then I get `quoted string split across lines` from checkpatch, should we keep a long line, or a checkpatch warning?

@aiChaoSONG You don't need to split quoted string if you take @ujfalusi 's suggestion. :)